### DISCUSSION TITLE
How viable is lapce for a hobbiest like me?

### DISCUSSION THREAD

**Discussion by zephiowl** [lapce/lapce#2104] [Q&A] [answered]
Hey so I'm mostly looking for people that have actually used this for a while to provide me some of the ups and downs when it comes to lapce and if it may be for me? 

I mostly code in rust; while I'm now also learning java. I also do some minor development in html, js, css, python, and lua. 
Do you personally believe that lapce would be a currently viable alternative for vscode x intelij (free) ides? I know its pre alpha and not perfect and probably has a lot of growing pains. And I'm willing to work trough them to support a cool project like this. I don't currently program for work and i mostly do it as a hobby in my free time with no real pressure or stress, Just to learn and have fun.

If you could tell me about your experience with the program and how it functions for you as well as key points i may or may not want to consider that would be great!

**Comment by Meiyo40** [comment, marked as answer]
Hello,

Since no one has responded, I'll give you my perspective.

As an hobbiest, you should focus on a stable environment to do things smoothly and quickly.
Especially if you are using different technology stacks.

For that, VSCode is great, it is well established, you have plenty of extensions to improve it, you don't waste time in settings, except maybe to choose your theme, hum....
Maybe CLion if you have money to spend or access to a license.

Lapce is a very interesting project and I hope to see it grow.
But still too young for a daily use with several stacks.
